### ISSUE TITLE
The bad classification performance of tuned model for Tiny Imagenet 

### ISSUE THREAD

**Issue by dubowsky** [hendrycks/pre-training#4]
Hi hendrycks,

Thanks for providing the finetuned models for the uncertainty task on Tiny Imagenet.

When I run the test.py using the snapshots/tune/wrn_baseline_epoch_19.pt under uncertainty/TinyImageNet. I found the test error is around 99.47% but it should be expected to be around 35% according to the csv log. So I wonder if there needs some preprocessing for the Tiny ImageNet dataset? Currently, I directly use the val part of Tiny Imagenet from https://image-net.org/download-images.php. The whole size of the Tiny Imagenet is 236 MB.

Many Thanks.

**Comment by dubowsky** [comment]
I could not access the instruction on imageNet before due to the website's error. The validation set should be preprocessed for ImageFolder to use.
